### ISSUE TITLE
[1.16.5] A feature that automatically adds the CanPlaceOn, CanBreak and CanHit tags to an item/block when crafted or obtained

### ISSUE THREAD

**Issue by ghost** [exaskye/Fabrication-Features#79]
I would like a feature that does that, it is really useful in rpg modpack/servers

**Comment by exaskye** [comment]
Do loot tables not meet your needs here?

**Comment by ghost** [comment]
Nope, I also want this feature to fix some bugs from a mod-pack like making some fluid buckets not placeable (It is supposed to be not placeable, these fluid buckets places a missing fluid block instead) using the CanPlaceOn, crashing when hitting an entity with a specific sword and I wanted to fix that using the CanHit tag... but the main reason is for RPG-like purposes, like crafting a weapon that would only work on Withers. Would really appreciate it to be added in the next 1.16.5 update :)
